ipytone/__init__.py

```diff
@@ -4,7 +4,7 @@
 # Copyright (c) Benoit Bovy.
 # Distributed under the terms of the Modified BSD License.
 
-from .ipytone import Oscillator
+from .ipytone import get_destination, Oscillator
```

Comment:
The convention with widgets is to have `tone.py` instead of `ipytone.py` here, but that's not a requirement.

Reply:
I'll fix that in #9